<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RGB Viewer v2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>39</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HBox">
     <item>
      <layout class="QVBoxLayout" name="VBox_red">
       <item>
        <layout class="QHBoxLayout" name="CenterVBox_red">
         <item>
          <widget class="QSlider" name="verticalSlider_red">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::handle:vertical {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 white, stop:1 red);
	border: 1px solid #999999;
	border-radius: 5px;
}</string>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_red">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VBox_green">
       <item>
        <layout class="QHBoxLayout" name="CenterVBox_green">
         <item>
          <widget class="QSlider" name="verticalSlider_green">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::handle:vertical {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 white, stop:1 green);
	border: 1px solid #999999;
	border-radius: 5px;
}</string>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_green">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VBox_blue">
       <item>
        <layout class="QHBoxLayout" name="CenterVBox_blue">
         <item>
          <widget class="QSlider" name="verticalSlider_blue">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::handle:vertical {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 white, stop:1 blue);
	border: 1px solid #999999;
	border-radius: 5px;
}</string>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_blue">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_red">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>16</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_green">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>16</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_blue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>16</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border: 1px solid;
}</string>
    </property>
    <property name="text">
     <string>COLOR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="listView_colorNames">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>141</width>
      <height>251</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_conserver">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Conserver</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HChoix">
     <item>
      <layout class="QVBoxLayout" name="VChoixG">
       <item>
        <widget class="QLabel" name="label_choix1">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border: 1px solid;
}</string>
         </property>
         <property name="text">
          <string>Choix 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_choix2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border: 1px solid;
}</string>
         </property>
         <property name="text">
          <string>Choix 2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_choix3">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border: 1px solid;
}</string>
         </property>
         <property name="text">
          <string>Choix 3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VChoixD">
       <item>
        <widget class="QLabel" name="label_choix4">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border: 1px solid;
}</string>
         </property>
         <property name="text">
          <string>Choix 4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_choix5">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border: 1px solid;
}</string>
         </property>
         <property name="text">
          <string>Choix 5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_choix6">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border: 1px solid;
}</string>
         </property>
         <property name="text">
          <string>Choix 6</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
